Gemfile

```diff
@@ -3,7 +3,13 @@ source 'https://rubygems.org'
 gem 'rake'
 gem 'minitest'
 gem 'minitest-reporters'
-gem 'http'
+gem 'minitest-focus'
+gem 'capybara'
+gem 'capybara_minitest_spec'
+# gem 'selenium-webdriver'
```

Comment:
I was briefly testing in selenium + chrome.  Yeah, I'll probably just nuke that commented out stuff.